napalm_nxos/nxos.py

```diff
@@ -204,7 +204,59 @@ def get_facts(self):
                 iface_list.append(intf)
 
         return results
+  
+    def get_fex_information(self):
+        item_list = list()
+
+        fex_command = 'show fex'
+
+        fex_raw_output = self.cli([fex_command]).get(fex_command, '')
```

Comment:
This command does not work on my switches (nexus 5k):

```bash
sw01.sfo04# show fex
                    ^
% Incomplete command at '^' marker.
```

Does ```show fex drops``` provide the same output?